src/auxiliary/containers.jl

```diff
@@ -50,282 +47,271 @@ length, the first `new_length` elements will be retained. If `new_length` is
 function remove_shift!(c::AbstractContainer, first::Int, last::Int)
-  @assert 1 <= first <= length(c) "First cell out of range"
-  @assert 1 <= last <= length(c) "Last cell out of range"
+    @assert 1<=first<=length(c) "First cell out of range"
+    @assert 1<=last<=length(c) "Last cell out of range"
 
-  # Return if removal would be a no-op
-  if last < first
-    return c
-  end
+    # Return if removal would be a no-op
+    if last < first
+        return c
+    end
 
-  # Delete connectivity of cells to be removed
-  delete_connectivity!(c, first, last)
+    # Delete connectivity of cells to be removed
+    delete_connectivity!(c, first, last)
 
-  if last == length(c)
-    # If everything up to the last cell is removed, no shifting is required
-    invalidate!(c, first, last)
-  else
-    # Otherwise, the corresponding cells are moved forward
-    move!(c, last + 1, length(c), first)
-  end
+    if last == length(c)
+        # If everything up to the last cell is removed, no shifting is required
+        invalidate!(c, first, last)
+    else
+        # Otherwise, the corresponding cells are moved forward
+        move!(c, last + 1, length(c), first)
+    end
 
-  # Reduce length
-  count = last - first + 1
-  c.length -= count
+    # Reduce length
+    count = last - first + 1
+    c.length -= count
 
-  return c
+    return c
 end
 remove_shift!(c::AbstractContainer, id::Int) = remove_shift!(c, id, id)
 
-
 # Remove cells and fill gap with cells from the end of the container (to reduce copy operations)
 function remove_fill!(c::AbstractContainer, first::Int, last::Int)
-  @assert 1 <= first <= length(c) "First cell out of range"
-  @assert 1 <= last <= length(c) "Last cell out of range"
+    @assert 1<=first<=length(c) "First cell out of range"
+    @assert 1<=last<=length(c) "Last cell out of range"
 
-  # Return if removal would be a no-op
-  if last < first
-    return c
-  end
+    # Return if removal would be a no-op
+    if last < first
+        return c
+    end
 
-  # Delete connectivity of cells to be removed and then invalidate them
-  delete_connectivity!(c, first, last)
-  invalidate!(c, first, last)
+    # Delete connectivity of cells to be removed and then invalidate them
+    delete_connectivity!(c, first, last)
+    invalidate!(c, first, last)
 
-  # Copy cells from end (unless last is already the last cell)
-  count = last - first + 1
-  if last < length(c)
-    move!(c, max(length(c) - count + 1, last + 1), length(c), first)
-  end
+    # Copy cells from end (unless last is already the last cell)
+    count = last - first + 1
+    if last < length(c)
+        move!(c, max(length(c) - count + 1, last + 1), length(c), first)
+    end
 
-  # Reduce length
-  c.length -= count
+    # Reduce length
+    c.length -= count
 
-  return c
+    return c
 end
 
-
 # Reset container to zero-length and with a new capacity
 function reset!(c::AbstractContainer, capacity::Int)
-  @assert capacity >=0
+    @assert capacity >= 0
```

Comment:
Strangely, here, the JuliaFormatter inserts a whitespace, but above there are no whitespaces around `<=` and similar in macros.

Reply:
Yes, that's strange 😕

Reply:
Well, the other `@assert`s have a message attached to them, don't they?